src/os/bluestore/BlueStore.h

```diff
@@ -1655,6 +1655,8 @@ class BlueStore : public ObjectStore,
     Sequencer *parent;
     BlueStore *store;
 
+    spg_t shard_hint;
```

Comment:
i think we can just remove the `shard_hint` in `ObjectStore::Sequencer` with this change. as nobody is actually reading it. `fio_ceph_objectstore.cc` sets it, but i am not sure if there is any consumer of this value.

Reply:
The OSD is also using this to pass the shard_hint through (see PG ctor).  So it goes PG -> Sequencer -> BlueStore::OpSequencer